shell/platform/tizen/flutter_tizen_shell.cc

```diff
@@ -0,0 +1,121 @@
+#include <assert.h>
+#include <unistd.h>
+#include <chrono>
+#include <iostream>
+#include <thread>
+
+#include "flutter/shell/platform/tizen/flutter_tizen_engine.h"
+#include "flutter/shell/platform/tizen/public/flutter_tizen.h"
+
+extern int gApp_width;
+extern int gApp_height;
+
+std::string TPK_ROOT_PATH = "/tpkroot";
+std::string LIB_PATH = "/lib";
+std::string RES_PATH = "/res";
+
+class FlutterApp {
+ public:
+  explicit FlutterApp() {}
+  virtual ~FlutterApp() {}
+
+  bool OnCreate() {
+    printf("Launching a Flutter application...\n");
+
+    std::string assets_path;
+    std::string icu_data_path;
+    std::string aot_lib_path;
+    FlutterDesktopEngineProperties engine_prop = {};
+    std::vector<const char*> switches;
+
+    std::string tpk_root;
+    if (app_path_.empty()) {
+      char path[256];
+      getcwd(path, sizeof(path));
+      tpk_root = path + TPK_ROOT_PATH;
+    } else {
+      tpk_root = app_path_;
+    }
+
+    assets_path = tpk_root + RES_PATH + "/flutter_assets";
+    icu_data_path = tpk_root + RES_PATH + "/icudtl.dat";
+    aot_lib_path = tpk_root + LIB_PATH + "/libapp.so";
+
+    switches.push_back("--disable-observatory");
+    switches.push_back("--verbose-logging");
+    switches.push_back("--enable-dart-profiling");
+    switches.push_back("--enable-checked-mode");
+
+    engine_prop.assets_path = assets_path.c_str();
+    engine_prop.icu_data_path = icu_data_path.c_str();
+    engine_prop.aot_library_path = aot_lib_path.c_str();
+    engine_prop.switches = switches.data();
+    engine_prop.switches_count = switches.size();
+    engine_ = reinterpret_cast<flutter::FlutterTizenEngine*>(
+        FlutterDesktopRunEngine(engine_prop, true));
+
+    if (!engine_) {
+      printf("Could not launch a Flutter application.\n");
+      return false;
+    }
+    // RegisterPlugins(this);
+    return true;
+  }
+  void OnResume() {}
+  void OnPause() {}
+  void OnTerminate() {
```

Comment:
This implementation seems difficult to have an app lifecycle.. What are your plans?

Reply:
There are no plans for these yet. 
These are just shell's stub that I want to look like the app's executable code (maybe for the future?)